src/common/import_session.c

```diff
@@ -307,14 +307,36 @@ const char *dt_import_session_filename(struct dt_import_session_t *self, gboolea
   return self->current_filename;
 }
 
+gboolean _dt_test_writable_dir(const char *path)
```

Comment:
I would move that into utility.h/c next to dt_util_test_image_file().

Reply:
good one.